.github/workflows/website-preview-destroy.yml

```
@@ -35,3 +39,19 @@ jobs:
         run: |
           aws sts get-caller-identity
           aws s3 rm s3://${{ env.S3_BUCKET_NAME }}/ --recursive --exclude "*" --include "pr-${{ env.PR_NUMBER }}/*"
+
+      # see https://dev.to/bobheadxi/branch-previews-with-google-app-engine-and-github-actions-3pco
+      - name: extract branch name
+        id: get_branch
+        shell: bash
+        env:
+          PR_HEAD: ${{ github.head_ref }}
+        run: echo "##[set-output name=branch;]$(echo ${PR_HEAD#refs/heads/} | tr / -)"
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:41: Double quote to prevent globbing and word splitting

(shellcheck)